txdbus/router.py

```diff
@@ -45,10 +45,8 @@ def match(self, m):
                     return
 
             if hasattr(self, 'path_namespace'):
-                # See https://github.com/cocagne/txdbus/issues/62
-                # FIXME: Skipping validation for this since we already have an
-                # open issue for it:
-                if m.path is None or not m.path.startswith(pn):  # noqa
+                if (m.path is None or
+                        not m.path.startswith(self.path_namespace)):
```

Comment:
The above style is the best one(tm) IMHO, because:

* Continuation lines are clearly distinguishable.
* Multiple statements are clearly separated. 

Glad we're on the same boat.